src/main/scala/io/tardieu/netwemo/checkers/HumidityChecker.scala

```diff
@@ -0,0 +1,45 @@
+package io.tardieu.netwemo.checkers
+import java.time.LocalTime
+
+import akka.actor.Props
+import com.typesafe.config.ConfigFactory
+import io.tardieu.netwemo.connectors.{NetatmoConnector, WemoConnector}
+
+import scala.concurrent.Future
+
+object HumidityChecker {
+
+  def props(wemoConnector: WemoConnector, netatmoConnector: NetatmoConnector): Props =
+    Props(new HumidityChecker(wemoConnector, netatmoConnector))
+
+}
+
+class HumidityChecker private (val wemoConnector: WemoConnector, val netatmoConnector: NetatmoConnector)
```

Comment:
Because `wemoConnector` and `netatmoConnector` are declared in the `Checker` trait.